resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -331,6 +332,18 @@ public DnsNameResolverBuilder ndots(int ndots) {
     }
 
     /**
+     * Set if domain / host names should be decoded to unicode when received.
+     * See <a href="https://tools.ietf.org/html/rfc3492">rfc3492</a>.
+     *
+     * @param decodeToUnicode if should get decoded
+     * @return {@code this}
+     */
+    public DnsNameResolverBuilder decodeToUnicode(boolean decodeToUnicode) {
```

Comment:
Don't like any of these... and also not like my name. Let me think a bit more about it :) Maybe we should just always convert for now and add a param later on ?